aws/load_balancer/cloudfront.tf

```diff
@@ -0,0 +1,65 @@
+locals {
+  s3_origin_id = "MaintenanceMode"
+}
+
+resource "aws_cloudfront_origin_access_identity" "maintenance_mode" {
+  comment = "Access Identity for the Maintenance Website"
+}
+
+resource "aws_cloudfront_distribution" "maintenance_mode" {
+  # checkov:skip=CKV_AWS_68: WAF ACL not required
+  # checkov:skip=CKV_AWS_86: Access logging not required
+  enabled             = true
+  http_version        = "http2"
+  default_root_object = "index.html"
+  web_acl_id          = aws_wafv2_web_acl.forms_acl.arn
+  aliases             = var.domains
```

Comment:
I have seen examples where this was not required. I plugged it because it was part of this in house implementation https://github.com/cds-snc/covid-alert-portal-production-terraform/pull/14

I would be curious to know if it works without it in our scenario.